examples/app.py

```diff
@@ -30,18 +30,87 @@
 .. code-block:: console
 
    $ cd examples
-   $ python app.py
+   $ flask -a app.py db init
+   $ flask -a app.py db create
+   $ flask -a app.py load_fixture
+   $ flask -a app.py --debug run
 """
 
 from __future__ import absolute_import, print_function
 
 from flask import Flask
+from flask_celeryext import FlaskCeleryExt
+from flask_cli import FlaskCLI
+from invenio_db import InvenioDB, db
+from invenio_pidstore import InvenioPIDStore
+from invenio_records import InvenioRecords
+from invenio_rest import InvenioREST
 
 from invenio_records_rest import InvenioRecordsREST
 
 # Create Flask application
 app = Flask(__name__)
+app.config.update(
+    CELERY_ALWAYS_EAGER=True,
+    CELERY_CACHE_BACKEND="memory",
+    CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
+    CELERY_RESULT_BACKEND="cache",
+)
+FlaskCLI(app)
+FlaskCeleryExt(app)
+InvenioDB(app)
+InvenioREST(app)
+InvenioPIDStore(app)
+InvenioRecords(app)
 InvenioRecordsREST(app)
 
+
+@app.cli.command()
+def load_fixture():
+    """Load test data fixture."""
+    import uuid
+    from invenio_records.api import Record
+    from invenio_pidstore.models import PersistentIdentifier, PIDStatus
+
+    # Record 1 - Live record
+    with db.session.begin_nested():
+        rec_uuid = uuid.uuid4()
+        pid1 = PersistentIdentifier.create(
+            'recid', '1', object_type='rec', object_uuid=rec_uuid,
+            status=PIDStatus.REGISTERED)
+        Record.create({'title': 'Registered '}, id_=rec_uuid)
+
+        # Record 2 - Deleted PID with record
+        rec_uuid = uuid.uuid4()
+        pid = PersistentIdentifier.create(
+            'recid', '2', object_type='rec', object_uuid=rec_uuid,
+            status=PIDStatus.REGISTERED)
+        pid.delete()
+        Record.create({'title': 'Live '}, id_=rec_uuid)
```

Comment:
Wouldn't it be more clear to create the record first and then mark the pid as deleted?